src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+
+  --link-hover: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  width: 100vw;
+  justify-content: space-between;
+}
+
+.logo__link {
+  margin-left: 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 50px;
+  height: 60px;
+  align-items: stretch;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: #000;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.nav__link:hover {
+  color: var(--link-hover);
+}
+
+.nav__link:not(:first-child) {
+  margin-left: 20px;
+}
+
+.is-active::after {
+  content: '';
+  position: relative;
+  top: 21px;
+  background-color: var(--link-hover);
+  border: 0.01px transparent var(--link-hover);
+  border-radius: 8px;
+  width: 38px;
+  height: 4px;
+}
+
+.is-active {
+  color: var(--link-hover);
+  display: flex;
+  flex-direction: column;
+  place-content: center center;
+}
+
+.is-active p {
```

Comment:
forget about using element type selectors. Excluding of course html and body element selectors.